src/sql/workbench/contrib/executionPlan/browser/executionPlanComparisonPropertiesView.ts

```diff
@@ -281,11 +281,11 @@ export class ExecutionPlanComparisonPropertiesView extends ExecutionPlanProperti
 				}
 				rows.push(row);
 				if (!isString(topProp.value) && !isString(bottomProp.value)) {
-					this.convertPropertiesToTableRows(topProp.value, bottomProp.value, rows.length - 1, indent + 2, rows);
```

Comment:
You can condense all this logic to the following since we only want to invoke convertPropertiestoTableRows when at least 1 or more arguments are not strings:
```
const topPropValue = !isString(topProp.value) ? topProp.value : undefined;
const bottomPropValue = !isString(bottomProp.value) ? bottomProp.value : undefined;
if (!(topPropValue !== undefined && bottomPropValue !== undefined)) {
    this.convertPropertiesToTableRows(topPropValue, bottomPropValue);
}